### PR TITLE
docs(core): add Nuxt recipe

### PR DESCRIPTION
**Pull request by barbados-clemens**
## Current Behavior

unsure how to setup nuxt app in an existing workspace

## Expected Behavior

content around setting up a nuxt app in a workspace.

## Related Issue(s)

blocked by: https://github.com/nrwl/nx-recipes/pull/53

Fixes #

### CODE REVIEW

File: docs/shared/recipes/add-stack/add-nuxt.md [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+
+and in the projects that are imported via `.vue` files can add the `scope:<app-name>` tag in the `project.json`:
+
+```json
+{
+  "tags": ["scope:<app-name>"]
+}
+```
+
+### Reexport Vue files
```
Comment (meeroslav) [inline docs/shared/recipes/add-stack/add-nuxt.md:127]:
```suggestion
### Re-export Vue files
```

File: docs/shared/recipes/add-stack/add-nuxt.md [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+
+TypeScript might give errors about not understanding exporting a vue file.
+
+```ts {% fileName="libs/ui/src/index.ts}
+export * from './lib/btn.vue';
+```
+
+in those cases you can create a `d.ts` file, saying `.vue` files are okay to export in ts files.
```
Comment (meeroslav) [inline docs/shared/recipes/add-stack/add-nuxt.md:135]:
```suggestion
In those cases you can create a `d.ts` file, saying `.vue` files are okay to export in ts files.
```

File: docs/shared/recipes/add-stack/add-nuxt.md [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+
+```json
+{
+  "tags": ["scope:<app-name>"]
+}
+```
+
+### Reexport Vue files
+
+TypeScript might give errors about not understanding exporting a vue file.
```
Comment (meeroslav) [inline docs/shared/recipes/add-stack/add-nuxt.md:129]:
```suggestion
TypeScript might give errors about not understanding exporting a `.vue` file.
```

**Review by meeroslav — CHANGES_REQUESTED** [review]
A few minor grammar suggestions

File: docs/shared/recipes/add-stack/add-nuxt.md [hunk 4]
```diff
@@ -0,0 +1,139 @@
...
+    "targets": {
+      "build": {
+        "outputs": ["{projectRoot}/.output", "{projectRoot}/.nuxt"]
+      }
+    }
+  }
+}
+```
+
+Make sure to ignore the `.output` and `.nuxt` folders from git and the Nx cache inputs.
```
Comment (isaacplmann) [inline docs/shared/recipes/add-stack/add-nuxt.md:50]: If you add `.output` and `.nuxt` to `.gitignore`, then you don't need to exclude them from inputs.  Nx will automatically ignore anything that git ignores.

File: docs/shared/recipes/add-stack/add-nuxt.md [hunk 5]
```diff
@@ -0,0 +1,139 @@
...
+  }
+
+  return alias;
+}
+
+export default defineNuxtConfig({
+  alias: getMonorepoTsConfigPaths('../../tsconfig.base.json'),
+  devtools: { enabled: true },
+});
+```
```
Comment (isaacplmann) [inline docs/shared/recipes/add-stack/add-nuxt.md:101]: Add some comments in here to show what is being modified in this file

File: docs/shared/recipes/add-stack/add-nuxt.md [hunk 6]
```diff
@@ -0,0 +1,139 @@
...
+}
+```
+
+and in the projects that are imported via `.vue` files can add the `scope:<app-name>` tag in the `project.json`:
+
+```json
+{
+  "tags": ["scope:<app-name>"]
+}
+```
```
Comment (isaacplmann) [inline docs/shared/recipes/add-stack/add-nuxt.md:125]: I don't know what you're telling people to do here.

**Review by isaacplmann — CHANGES_REQUESTED** [review]
Some inline comments.  You also need to add a section showing how to make a library and import it.

File: docs/shared/recipes/add-stack/add-nuxt.md (hunk 6 above)
Comment (barbados-clemens) [reply docs/shared/recipes/add-stack/add-nuxt.md:125]: this is building the graph via the tags and implicit dependencies, it's a second part to the stuff above this.

File: docs/shared/recipes/add-stack/add-nuxt.md (hunk 4 above)
Comment (barbados-clemens) [reply docs/shared/recipes/add-stack/add-nuxt.md]: I just changed it to say move it to the top level dir so nx ignores it.

**Review by isaacplmann — APPROVED** [review]

**Review by meeroslav — APPROVED** [review]

**Pull request merged**